### PR TITLE
[3052] - Add Simplecov gem

### PR DESCRIPTION
**Pull request by TobyRet**
### Context
- Adds SimpleCov gem
- Threshold set to 97% (currently coverage is 97.12)

### Guidance to review
- Please suggest any other setting you think should be added

### CODE REVIEW

**Comment by TobyRet** [comment]
[Add test coverage](https://trello.com/c/s1dYZLS1/3053-add-test-coverage)

**Comment by timabell** [comment]
I misunderstood the PR title, I suggest rewording it

**Comment by timabell** [comment]
It makes it sound like there was no tests and you added some :laughing: 

how about "Add SimpleCov gem"
